src/devices_models/devices/hydro_generation.jl

```diff
@@ -545,13 +547,13 @@ function nodal_expression!(
         include_parameters(
             psi_container,
             ts_data_active,
-            UpdateRef{H}(ACTIVE_POWER, "get_rating"),  # TODO: fix in PR #316
+            UpdateRef{H}(ACTIVE_POWER, "get_activepower"),
```

Comment:
Same logic as before 

Case 1: Parameters and Forecast `UpdateRef{H}(ACTIVE_POWER, "get_maxactivepower")`
Case 2: Parameters and No - Forecast `UpdateRef{H}(ACTIVE_POWER, "get_activepower")`